components/ReadingChallenge/ReadingChallenge.js

```diff
@@ -59,43 +64,57 @@ export default function ReadingChallenge({
   return (
     <StyledBody>
       <StyledBox>
-        {challenge !== null ? (
+        {challenge.type ? (
           <Paragraph>
-            You have read {progress} out of {challenge.amount} {challenge.type}
+            You have read {progress} out of {challenge.amount} {challenge.type}.
           </Paragraph>
         ) : (
-          <Paragraph>You have not created a challenge yet</Paragraph>
+          <Paragraph>You have not created a challenge yet.</Paragraph>
         )}
-        <StyledButton onClick={openModal}>
-          {challenge !== null ? "🖊️" : "+"}
-        </StyledButton>
+        <StyledEditButton onClick={openModal}>
+          {challenge !== null ? <EditIcon /> : "+"}
+        </StyledEditButton>
```

Comment:
I guess you can simplify the condition. ➡️  If challenge exists is **truthy**.

```suggestion
        <StyledEditButton onClick={openModal}>
          {challenge ? <EditIcon /> : "+"}
        </StyledEditButton>
```